src/components/header/ConnectWalletButton.tsx

```diff
@@ -11,16 +11,15 @@ import { mapConnectorNameToIcon } from './ConnectorIconMap';
 import { Text } from '../common/Typography';
 
 export type ConnectWalletButtonProps = {
+  accountData: any;
+  disconnect: () => void;
   buttonStyle?: 'secondary' | 'tertiary';
 };
 
 export default function ConnectWalletButton(props: ConnectWalletButtonProps) {
+  const { accountData, disconnect } = props;
```

Comment:
It might be cleaner to not have these props. Just have the `ConnectWalletButton` fetch its own `accountData` using `useAccount` like before. This could result in some duplicate state, but it prevents us from having to put a bunch of `useAccount`s elsewhere in the codebase. What do you think?

Reply:
Referenced in issue #79. Resolving conversation for now.